tools/data-handler/src/containers/project.ts

```diff
@@ -233,6 +233,34 @@ export class Project extends CardContainer {
     return resources;
   }
 
+  /**
+   * Returns a new unique card key with project prefix (e.g. test_x649it4x).
+   * @returns a new card-key string
```

Comment:
Super minor; just fixing the naming, so use "card key" instead of "card-key" or "cardkey" when talking to humans (like here).

Reply:
Fixed in [852fc33](https://github.com/CyberismoCom/cyberismo/pull/127/commits/852fc3324963ad17dac30dd7baa1091d6b0f5c39)